src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -1718,7 +1718,35 @@ public final class JavadocTokenTypes {
     /** End table row tag. */
     public static final int TR_TAG_END = JavadocParser.RULE_trTagEnd + RULE_TYPES_OFFSET;
 
-    /** Table cell html tag: {@code <td></td>}. */
+    /**
+     * Table cell HTML tag.
+     *
+     * <p><b>Example:</b></p>
+     * <pre>{@code <td>Cell Content</td>}</pre>
+     * <b>Tree:</b>
+     * <pre>
+     * {@code
+     *   JAVADOC -> JAVADOC
+     *        |--NEWLINE -> \r\n
+     *        |--LEADING_ASTERISK ->  *
+     *        |--TEXT ->
+     *        |--HTML_ELEMENT -> HTML_ELEMENT
+     *        |   `--TD -> TD
+     *        |       |--TD_TAG_START -> TD_TAG_START
+     *        |       |   |--START -> &lt;
+     *        |       |   |--TD_HTML_TAG_NAME -> td
+     *        |       |   `--END -> &gt;
+     *        |       |--TEXT -> Cell Content
```

Comment:
minor comment, based on other examples, it looks like we avoid adding any additional content for `html` tags, but I do not consider it as something significant

Otherwise LGTM

@romani cc

Reply:
Some text is good, as in real life all tag has text.